src/views/camp/camp.scss

```diff
@@ -0,0 +1,82 @@
+@import "../../colors";
+@import "../../frameless";
+
+#view {
+    padding: 0;
+}
+.topImg {
+    margin-top: -10px;
+    padding: 0;
+    width: 100%;
+}
+.bubbles {
+    display: block;
+    margin: 0 auto;
+    margin-top: .938em;
+    width: 45%;
+}
+h3 {
+    border-bottom: 1px solid $ui-border;
+}
+.flex-row {
+    &.sidebar-row {
+        align-items: flex-start;
+
+        .body-copy {
+            width: $cols5;
+        }
+
+        .sidebar {
+            width: $cols5;
+        }
+    }
+}
+#particpate {
+    position: relative;
+    
+    h2, h3, p {
+        color: $ui-white;
+    }
+}
+#info {
+    h2, p {
+        color: $ui-white;
+    }
+}
+.info-content {
+    div {
+        max-width: $cols4;
+        text-align: center;
+    }
+}
+.infoImg {
```

Comment:
I think these need to have a fixed height. Right now, the text is not horizontally aligned because of the different sizes of the images:
<img width="975" alt="screen shot 2017-07-18 at 9 08 53 am" src="https://user-images.githubusercontent.com/1815772/28318495-0d7d2166-6b99-11e7-8105-468e8623d523.png">
I actually kinda like the cascading effect of the text gradually moving down, but I imagine that we should try to align them :). When I add a fixed height of `height: 120px;`, it looks like this:
<img width="1031" alt="screen shot 2017-07-18 at 9 12 58 am" src="https://user-images.githubusercontent.com/1815772/28318598-629704f0-6b99-11e7-8bf1-227a0ff0f4cb.png">